cloud/volume.go

```diff
@@ -31,6 +31,8 @@ func CreateVolume(ctx context.Context, env evergreen.Environment, volume *host.V
 
 func GetEC2ManagerForVolume(ctx context.Context, vol *host.Volume) (Manager, error) {
 	provider := evergreen.ProviderNameEc2OnDemand
+	// WARNING: We unfortunately have to hard-code variables for E2E testing.
+	// Note that this should be avoided when possible, but is necessary in this case.
```

Comment:
Can we also create consts for `SETTINGS_OVERRIDE` and `7f909d47566126bd39a05c1a5bd5d111c2e68de3830a8be414c18c231a47f4fc` with a comment on each one explaining what it is?

Reply:
i named the variable to be self explainatory so that we wouldn't need a comment! 
left SETTINGS_OVERRIDE alone because our codebase uses that as a string literal pretty often